src/components/results/result.vue

```diff
@@ -32,7 +32,7 @@
         {{ result.name }}
       </h2>
       <span v-if="showDescription" class="x-text">{{ result.season }}</span>
-      <div class="x-list x-list--horizontal x-list--gap-04">
+      <div class="x-list x-list--horizontal x-list--wrap x-list--justify-center x-list--gap-03">
```

Comment:
The gap between the result new and old price should be 12px (our 04 option). Why did you decided to change it?
<img width="383" alt="image" src="https://user-images.githubusercontent.com/68222542/179717606-6a15c3f1-b3fa-432d-8d4c-9dfcbba390bc.png">

Reply:
With this change the result price is always centered, and it shouldn't
<img width="922" alt="image" src="https://user-images.githubusercontent.com/68222542/179718141-6988533b-579f-4245-8fbb-8f998e37917c.png">

Reply:
> The gap between the result new and old price should be 12px (our 04 option). Why did you decided to change it? <img alt="image" width="383" src="https://user-images.githubusercontent.com/68222542/179717606-6a15c3f1-b3fa-432d-8d4c-9dfcbba390bc.png">

The change for the gap was done because when the prices wrapped the gap between them was too big. Talked again with Cristina to confirm and she gave the OK to the gap change from `-04` to `-03`.

> With this change the result price is always centered, and it shouldn't <img alt="image" width="922" src="https://user-images.githubusercontent.com/68222542/179718141-6988533b-579f-4245-8fbb-8f998e37917c.png">

Mixup with Kenay, there the price is center, will fix it.